filebeat/docs/modules/nats.asciidoc

```diff
@@ -0,0 +1,60 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-nats]]
+:modulename: nats
+:has-dashboards: true
+
+== nats module
+
+This is the nats module.
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+TODO: document with what versions of the software is this tested
+
+
+include::../include/running-modules.asciidoc[]
+
+[float]
+=== Example dashboard
+
+This module comes with a sample dashboard. For example:
+
+TODO: include an image of a sample dashboard. If you do not include a dashboard,
```

Comment:
Can you check all the TODO's in here? If there is no screenshot yet, remove the todo. Same for the the other TODO